core/include/utility/Exception.hpp

```diff
@@ -16,9 +16,14 @@ namespace Utility
 		Unknown_Exception          = Exception_Unknown_Exception,
         Non_existing_Image         = Exception_Non_existing_Image,
         Non_existing_Chain         = Exception_Non_existing_Chain
+        // TODO: from Chain.cpp
+        // Last image deletion ?
+        // Empty clipboard     ?
```

Comment:
As long as the resulting log messages will be equivalently clear that would be the right thing to do. In the cases you mention that may not be possible, I'm not quite sure right now.